### PR TITLE
Add sponsors for Sep-16 announcement.

### PR DESCRIPTION
**Pull request by stylesen**

### CODE REVIEW

**Comment by bravegnu** [comment]
Can we drop the white background from Toyota Connected? This needs to be done for MavenHive too. This need not block merge. But it would be great if we can get this done before the merge.

**Comment by stylesen** [comment]
> Can we drop the white background from Toyota Connected? This needs to be done for MavenHive too. This need not block merge. But it would be great if we can get this done before the merge.

This is how the sponsors have provided the logo (with white background) I couldn't edit it, probably I shall ask for a logo with transparent background to them.

**Review by bravegnu — APPROVED** [review]

**Pull request merged**
